components/client/src/main/java/com/hotels/styx/client/applications/metrics/OriginMetrics.java

```diff
@@ -40,114 +36,95 @@
  * connections scheduled on different event loops.
  */
 public class OriginMetrics implements OriginStats {
-    private static final Logger LOG = LoggerFactory.getLogger(OriginMetrics.class);
-    private static final int SERVER_ERROR_CLASS = 5;
+    public static final String ORIGIN_TAG = "origin";
+    public static final String APP_TAG = "appId";
+    public static final String STATUS_TAG = "statusCode";
+    public static final String STATUS_CLASS_TAG = "statusClass";
 
-    private final ApplicationMetrics applicationMetrics;
+    public static final String SUCCESS_COUNTER_NAME = "request_success_count";
+    public static final String FAILURE_COUNTER_NAME = "request_error_count";
+    public static final String STATUS_COUNTER_NAME = "response_status_count";
```

Comment:
Probably worth talking about a common approach to naming. Micrometer recommends dots for tag and meter names. 
Also there are various commonalities in names. The App and Origin tags are used in #672 and HTTP status codes (possibly as tags) in #668. The idea of a status class might also be useful in #668.